jlm/tooling/Command.cpp

```diff
@@ -449,7 +449,7 @@ LlvmOptCommand::ToString(const Optimization & optimization)
 {
   static std::unordered_map<Optimization, const char*>
     map({
-          {Optimization::Mem2Reg, "-mem2reg"},
+          {Optimization::Mem2Reg, "-passes=mem2reg"},
```

Comment:
No, this is not the right place to insert the "-passes=". It should happen in the ToString() method.

Reply:
Moved to ToString().